### ISSUE TITLE
Staging solution for (markdown) updates

### ISSUE THREAD

**Issue by Maartenvm** [NLeSC/software.esciencecenter.nl#213]
We cannot have the production version go down every time there is an unforeseen error. We need to check the site in a safe environment before we push the updates to the live version.

**Comment by jspaaks** [comment]
Agree. Note that the safe environment would include both new code for the site as well as new data (the last time we had trouble it came from the data/content side of things, the site itself worked fine provided the input data was correct.)
